### DISCUSSION TITLE
Перенос строки в заголовке поля на быстрой форме.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#498] [Как это сделать?] [answered]
Есть возможность сделать перенос строки в заголовке поля на быстрой форме?

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/d6041ad6-abed-451a-a25b-d54f6edf0d79)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Никак
